packages/round-manager/src/features/round/FundContract.tsx

```diff
@@ -92,7 +92,9 @@ export default function FundContract(props: {
     props.round &&
     payoutTokens.filter(
       (t) =>
-        t.address.toLocaleLowerCase() == props.round?.token?.toLocaleLowerCase()
+        t.address.toLocaleLowerCase() ==
```

Comment:
Not introduced by this PR but I think we should use `toLowerCase` here, locale isn't relevant for chain addresses.

Reply:
@bard yes it makes sense I updated it